platform/broadcom/platform-modules-nokia.mk

```diff
@@ -12,3 +12,7 @@ $(NOKIA_IXR7250_PLATFORM_MODULE)_PLATFORM = x86_64-nokia_ixr7250e_sup-r0
 $(NOKIA_IXR7250_PLATFORM_MODULE)_PLATFORM += x86_64-nokia_ixr7250e_36x400g-r0
 SONIC_DPKG_DEBS += $(NOKIA_IXR7250_PLATFORM_MODULE)
 
+NDK_VERSION = 21.12.20
+NOKIA_NDK_DEBIAN = ndk_$(NDK_VERSION)_amd64.deb
+$(NOKIA_NDK_DEBIAN)_URL = "https://github.com/Nokia-ION/ndk-releases/releases/download/v$(NDK_VERSION)/$(NOKIA_NDK_DEBIAN)"
```

Comment:
this is a binary blob, can you put into msft repo?

Reply:
Sure will do it on msft repo -- Closing this PR.